src/bro/conversation.py

```diff
@@ -1,21 +1,26 @@
+import base64
 import json
 import logging
+import os.path
 from dataclasses import dataclass
-from typing import Any
+from pathlib import Path
+from typing import Any, List
```

Comment:
```suggestion
from typing import Any
```

Use `list` instead; the aliases from `typing` are deprecated. There is a PEP with the details, you can look it up later.